test/e2e/tests/petnames-helpers.js

```diff
@@ -26,7 +26,7 @@ async function clickName(driver, value) {
   });
 }
 
-async function saveName(driver, value, name, proposedName) {
+async function saveName(driver, value, name, proposedName, isUpdate = false) {
```

Comment:
Am I right these are changes from the previous PR?

I think best practice for chained PRs is to create your PR targetting the previous PR to ensure the most minimal diff, and then when the previous PR is merged the base will automatically be changed to `develop`.

Reply:
Apologies for the confusion. These changes should be in develop now.